UIKitHw/Extensions/Datasource+Extension.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Datasource+Extension.swift
+//  UIKitHw
+//
+//  Created by Aleksandr Dorofeev on 04.10.2022.
+//
+
+import UIKit
+
+/// UIPickerViewDataSource.
+extension BookViewController: UIPickerViewDataSource {
+  func numberOfComponents(in pickerView: UIPickerView) -> Int {
+    let numberOfComponents = 1
```

Comment:
```suggestion
    1
```
вместо обеих строчек